docs/docs/statistics/quantile.mdx

```diff
@@ -0,0 +1,96 @@
+
+## How do I interpret quantile test results?
+
+Suppose you are running an A/B test designed to reduce website latency. Your effect estimate and 95% confidence interval (in milliseconds) for latency at P99 is -7 ms (-9 ms, -5 ms). How do you interpret this?
+
+Consider one universe where **all** customers received control (not just the customers assigned to control). In this universe, website latency for 99% of customers is no more than 145 ms. That is, P99 for control is 145 ms.
+
+Consider another universe where **all** customers are assigned to treatment. In the treatment universe, website latency for 99% of customers is no more than 139 ms. So the true effect at P99 is 139 ms - 145 ms = -6 ms.
+
+A quantile test tries to estimate this difference. You interpret the interval above as “There is a 95% chance that the difference in P99 latencies across the groups is in (-9ms, -5ms)”.
+
+## Should I aggregate by experiment user before taking quantile?
+
+Below we describe how quantile testing differs in event- vs user-level analyses. Suppose that your new feature is designed to lower webpage request times. If you want to reduce the largest request times across all web sessions, then use quantile testing for event-level data. This can help you learn if your new feature reduced the 99th percentile of request times. If you want to reduce total request times for your most frequent customers, then use quantile testing for user-level data. Here Growthbook sums the total request times for all events within a customer, then compares percentiles of these sums across variations.
+
+## How do I run a quantile test in GrowthBook?
+
+Running a quantile test is just as easy as running a mean test. Currently, quantile testing is available only for .
+
+1. Navigate to your [Fact Table](/app/fact-tables) and select "Add Metric".
+2. Select “Quantile” from “Type of Metric”.
+3. Toggle “Aggregate by Experiment User before taking quantile” if you want to compare quantiles across variations at the user granularity, after summing row values at the user level. The default is at the event granularity.
+4. Pick your quantile level from the defaults (p50, p90, p95, p99) or use a custom value. Guidance describing the range of available values is in our [FAQ](#faq).
+5. Decide whether you want zeros to be included in the analysis (see [FAQ](#faq) below).
+6. Select your metric window as you would for a mean test.
+7. Submit!
+
+![User interface for quantile metrics](/images/statistics/quantile.png)
+
+## Growthbook implementation
```

Comment:
Maybe move this whole section below the FAQ, TBH.

Reply:
done, good call